compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1474,20 +1474,22 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
     /// Given a function definition like:
     ///
     /// ```rust
+    /// use std::fmt::Debug;
+    ///
     /// fn test<'a, T: Debug>(x: &'a T) -> impl Debug + 'a {
     ///     x
     /// }
     /// ```
     ///
     /// we will create a TAIT definition in the HIR like
     ///
-    /// ```
+    /// ```rust,ignore (pseudo-Rust)
```

Comment:
If it was compiling before, why not add a hidden import instead?

Reply:
> why not add a hidden import instead?

There are multiple errors, and one of them was:
```
error: unconstrained opaque type
 --> src/lib.rs:1488:30
  |
4 | type TestReturn<'a, 'x, T> = impl Debug + 'x;
  |                              ^^^^^^^^^^^^^^^
  |
  = note: `TestReturn` must be used in combination with a concrete type within the same item
```